docker-compose.yml

```diff
@@ -21,6 +21,11 @@ services:
     - type: bind
       source: ./testdata/pki/cassandra.key
       target: /etc/scylla/db.key
+    healthcheck:
```

Comment:
This means it would take 5 minutes for Docker to claim Scylla unhealthy.

Reply:
I followed the healthchecks in the Rust driver (https://github.com/scylladb/scylla-rust-driver/blob/main/test/cluster/docker-compose.yml#L25), do you suggest other values in the healthcheck?